components/create_post/create_post.jsx

```diff
@@ -536,21 +544,48 @@ class CreatePost extends React.PureComponent {
         const {
             currentChannel: updateChannel,
             userIsOutOfOffice,
+            allowReferencedGroups,
+            channelMemberCountsByGroup,
+            currentChannelMembersCount
         } = this.props;
 
-        const currentMembersCount = this.props.currentChannelMembersCount;
         const notificationsToChannel = this.props.enableConfirmNotificationsToChannel && this.props.useChannelMentions;
+        let memberCount = 0;
+        let timezoneCount = 0;
+        let groups = [];
```

Comment:
small thing but you could just rename these 

```suggestion
        let memberNotifyCount = 0;
        let channelTimezoneCount = 0;
        let mentions = [];
```
and then 

```
this.setState({
                channelTimezoneCount,
                memberNotifyCount,
                mentions,
                showConfirmModal: true,
            });
```

Reply:
Done